libs/gamify-api/apps/src/lib/apps.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@nestjs/common';
+import { App, ApplicationType } from '@coders/gamify-api/core';
+
+@Injectable()
+export class AppsService {
+
+    private apps: App[] = [];
+
+    constructor() {
+        this.mockApps();
+    }
+
+    getApps(): App[] {
```

Comment:
Something to consider: use a more generic (predictable) method name, like `getAll` or `findAll`?

Reply:
Optional :-)